packages/site-server/tsconfig.json

```diff
@@ -3,8 +3,10 @@
   "compilerOptions": {
     "rootDir": "./src",
     "noEmit": true,
-    "tsBuildInfoFile": "tsconfig.tsbuildinfo"
-  },
+    "tsBuildInfoFile": "tsconfig.tsbuildinfo",
+    "module": "esnext",
```

Comment:
Do we want to standardize everything on `esnext` and `node`? I'm a little hazy on exactly what the differences are between those and `nodenext`.

Reply:
I guess? I think the difference has to do with package exports and ESM vs CommonJS. I just know that this works now here... not whether or for how long we should avoid the Node* settings (Node16 didn't work either).